plugins/net_plugin/net_plugin.cpp

```diff
@@ -1150,7 +1150,7 @@ namespace eosio {
 
    // called from connection strand
    void connection::check_heartbeat( tstamp current_time ) {
-      if( protocol_version >= heartbeat_interval && latest_msg_time > 0 ) {
+      if( latest_msg_time > 0 ) {
```

Comment:
Not related to this PR. The version names of net_plugin should have a prefix of protocol or version. When I first read `protocol_version >= heartbeat_interval`, I thought `protocol_version` and `heartbeat_interval` were different units, how could they compare? Then found out `heartbeat_interval` talked about a version, not an interval value.

Reply:
Some already had `proto_` so added that to all of them.